src/spender/paths/spender@expenses@check_policies.yaml

```diff
@@ -0,0 +1,48 @@
+post:
+  tags:
+    - Expenses
+  summary: Check expense for policy violation
+  description: |
+    This API checks an expense for policy violations
+  operationId: check_policies_post
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: '../../components/schemas/expense.yaml#/spender_expense_check_policies_in'
```

Comment:
Remove `spender_` from `spender_expense_check_policies_in`
Or is there any specific reason?